formats/financial_releases_index/publisher/details.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "title",
+     "image",
+     "press_releases"
+   ],
+  "properties": {
+     "title": {
+       "type": "string"
+     },
+    "image": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "alt_text": {
+          "type": "string"
+        }
+      }
+    },
+    "press_releases": {
```

Comment:
@boffbowsh we have moved press releases into a links hash. Please could you review when you have time. We also have a couple of questions regarding referencing links inside html strings -would be great if we could chat to you about this briefly this morning if you have time? Thank you